components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtilTest.java

```diff
@@ -342,10 +356,25 @@ public void testGetUserConsentURL(Object oAuth2ParamObject, boolean isOIDC, bool
                     "is not found in url");
             Assert.assertTrue(ArrayUtils.contains(scopeArray, "internal_login"), "internal_login " +
                     "scope parameter value is not found in url");
+
             if (queryString != null && cacheEntryExists) {
                 Assert.assertTrue(consentUrl.contains(queryString), "spQueryParams value is not found in url");
             }
 
+            if (parameters.getScopes().contains("openid")) {
+                String requestedClaimString = URLDecoder.decode(consentUrl, "UTF-8");
+                int checkIndex = requestedClaimString.indexOf(REQUESTED_OIDC_SCOPES);
+                Assert.assertTrue(checkIndex != -1, "Requested OIDC scopes query parameter is not found in url.");
```

Comment:
Here although the assertion gives the impression we are checking for the existence of requested_oidc_scopes query string it seems we are actually checking for `requested_oidc_scopes=openid+profile`. Lets check for `requested_oidc_scopes=`